common/lib/common-definitions/api-extractor.json

```
@@ -1,4 +1,12 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "extends": "@fluidframework/build-common/api-extractor-common.json"
+  "extends": "@fluidframework/build-common/api-extractor-common-report.json",
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "<projectFolder>/api-report/"
+  },
+  "docModel": {
+    "enabled": true,
+    "apiJsonFilePath": "<projectFolder>/../../_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
```

I was wrong about the paths here. This can also be project-relative because the `build:docs` script copies the files to a central location.

```suggestion
    "apiJsonFilePath": "<projectFolder>/_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
```